### ISSUE TITLE
Command window spams with "QSocketNotifier: Multiple socket notifiers for same socket..."

### ISSUE THREAD

**Issue by pgozz** [spyder-ide/spyder#13555]
### What steps will reproduce the problem?

On my windows machine with Python 3.6, this happens everytime I enter anything in the command window.

## Versions

* Spyder version: 4.1.4 
* Python version: 3.6.8
* Qt version: 5.12.9
* PyQt5 version: 5.12.3
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.5.0 (OK)
diff_match_patch >=20181111    :  20200713 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.16.1 (OK)
jedi =0.17.1                   :  0.17.1 (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  1.1.0 (OK)
paramiko >=2.4.0               :  2.7.1 (OK)
parso =0.7.0                   :  0.7.0 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.0 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=1.0                   :  2.5.3 (OK)
pyls >=0.34.0;<1.0.0           :  0.34.1 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  0.7.2 (OK)
qtconsole >=4.6.0              :  4.7.5 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
sphinx >=0.6.6                 :  3.1.2 (OK)
spyder_kernels >=1.9.2;<1.10.0 :  1.9.2 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  19.0.1 (OK)

# Optional:
cython >=0.21                  :  None (OK)
matplotlib >=2.0.0             :  3.1.3 (OK)
numpy >=1.7                    :  1.18.5 (OK)
pandas >=0.13.1                :  0.25.3 (OK)
scipy >=0.17.0                 :  1.2.0 (OK)
sympy >=0.7.3                  :  1.5 (OK)
```

**Comment by ccordoba12** [comment]
Hey @pgozz, thanks for reporting. This error was fixed in one of our dependencies called `ipykernel`. So please update to its latest version (5.3.4) and let us know if it works for you.

**Comment by adriesse** [comment]
Do you mean version 1.9.3?

**Comment by ccordoba12** [comment]
Closing due to lack of response.

**Comment by pgozz** [comment]
@ccordoba12 thank your fast reply and apologies for my delayed response.  Updating ipykernel does seem to have fixed the issue, so closing was the right thing to do 👍

**Comment by ccordoba12** [comment]
Great! Thanks for the confirmation @pgozz!

**Comment by EricThomson** [comment]
This has started happening for me with Spyder 4.2+/ipykernel 5.5.5 on Windows 10. It is not an issue when I downgrade to Spyder 4.1, but I like the apply linter on save functionality available with Spyder 4.2+. 

Not sure if worth starting a new issue with this, but just wanted to flag it here.
